bitnami/harbor/README.md

```diff
@@ -255,6 +255,27 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `clair.podAnnotations`                                                      | Annotations to add to the clair pod          | `{}`        |
 | `clair.livenessProbe`                                                       | Liveness probe configuration                 | `Check values.yaml file` |
 | `clair.readinessProbe`                                                      | Readiness probe configuration                | `Check values.yaml file` |
+| **Notary**                                                                  |
```

Comment:
There are variables here that are not used

Reply:
true, I just reviewed them one by one :)